testsuite/standalone/src/test/java/org/wildfly/core/test/standalone/mgmt/BasicOperationsUnitTestCase.java

```diff
@@ -112,6 +115,38 @@
     @Inject
     private static ManagementClient managementClient;
 
+    @BeforeClass
+    public static void reset() throws IOException {
+        // clear the content-dir in case anything else has deployed there and not cleaned up.
+        ModelNode address = new ModelNode();
+        address = new ModelNode();
+        address.add("core-service", "server-environment");
+        address.protect();
+        ModelNode operation = new ModelNode();
+        operation.get(OP).set("path-info");
+        operation.get(OP_ADDR).set(address);
+        ModelNode result = managementClient.getControllerClient().execute(operation);
+
+        if (Operations.readResult(result).get("content-dir").get("resolved-path").isDefined()) {
+            String path = Operations.readResult(result).get("content-dir").get("resolved-path").asString();
+            if (path == null) {
+                return;
+            }
+            File file = new File(path);
+            if (!file.exists() || !file.isDirectory()) {
+                return;
+            }
+
+            for (File f : file.listFiles()) {
+                if (f.isFile()) {
+                    f.delete();
+                } else if (f.isDirectory()) {
+                    FileUtils.deleteDirectory(f);
+                }
+            }
+        }
+    }
```

Comment:
Ah, OK, I'd interpreted this as something the test really wanted to verify. I'll remove that and my last change and squash.